src/open_inwoner/react/components/SideNav/SideNav.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from 'react'
+import { SideNavigation } from '@gemeente-denhaag/side-navigation'
+import { MaterialIcon } from '../MaterialIcons/MaterialIcons'
```

Comment:
Small thing... might need to check this
```suggestion
import { MaterialIcon } from '@react/components/MaterialIcons/MaterialIcons'
```